Vagrantfile

```diff
@@ -73,6 +73,18 @@ Vagrant.configure(2) do |config|
     winw1.vm.host_name = "winw1"
     winw1.vm.box = "sig-windows-dev-tools/windows-2019"
     winw1.vm.box_version = "1.0"
+
+    winw1.vm.communicator = "winrm"
+    winw1.vm.guest = :windows
+    winw1.ssh.shell = "powershell"
+
+    winw1.winrm.username = "vagrant"
+    winw1.winrm.password = "vagrant"
```

Comment:
FYA, this is present in https://github.com/kubernetes-sigs/sig-windows-dev-tools/blob/master-windows-native/Vagrantfile